### ISSUE TITLE
找不到demo.py的输出

### ISSUE THREAD

**Issue by IndowK** [YoungSeng/Interspeech2022#13]
作者您好
我在运行demo.py之后，命令行输出：
adversarial classifier for content embedding
common classifier for speaker embedding
/root/miniconda3/envs/myconda/lib/python3.8/site-packages/torch/nn/functional.py:1805: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
source_target_e.ne.n_U
100%|███████| 47360/47360 [10:16<00:00, 76.88it/s]

demo.py中输出目录应该是这行代码result_path = "/mnt/SRD-VC-master/My_model/my_demo"，但是我在对应的文件夹下并没有找到任何输出，我想请问您一下这是为什么

**Comment by YoungSeng** [comment]
我建议仔细检查一下代码看看保存到哪里了，我这里测试是没有问题的

![image](https://user-images.githubusercontent.com/37477030/231935260-daca7372-84aa-4763-87b9-c7ba62ec7458.png)
